pkg/kubectl/cmd/label.go

```diff
@@ -18,11 +18,12 @@ package cmd
 
 import (
 	"fmt"
+	"github.com/spf13/cobra"
```

Comment:
nit: move this back to the second group of imports. The idea is keeping a group of standard library imports separate from third party libs.